src/components/Button/index.jsx

```diff
@@ -83,6 +84,7 @@ const Button = forwardRef(
           'transition--default',
           `fdsButton--${size}`,
           `fdsButton--${theme}`,
+          `margin--${margin}`,
```

Comment:
I either need to do this:

```
...(cond ? ['a'] : [])
```

or create some memoized function and execute it here. Basically this will leave `margin--undefined` on any button that isn't using margin.